lib/ansible/plugins/doc_fragments/action_core.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) , Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+# WARNING: this is mostly here as a convinence for documenting core behaviours, no plugin outside of ansbile-core should use this file
+class ModuleDocFragment(object):
+
+    # Standard documentation fragment
+    DOCUMENTATION = r'''
+attributes:
+    async:
+      support: none
+    become:
+      support: none
+    core:
+      description: This is a 'core engine' feature and is not implemented like most task actions, so it is not overridable in any way via the plugin system.
+      support: full
+    connection:
+      support: none
+    platform:
+      platforms: all
+    until:
+      description: Denotes if this action objeys until/retry/poll keywords
+      support: full
+    tags:
+      description: Allows for the 'tags' keyword to control the selection of this action for execution
+      support: full
+'''
+
+    IMPORT = r'''
+attributes:
+    bypass_task_loop:
+      description: Denotes if the action itself will be conditioned on the ``when`` keyword
+      details: the task itself is not looped, but the loop is applied to each imported task
+      support: partial
+    conditional:
+      details: The task itself ignores the conditional but adds it to each task imported
```

Comment:
... and this be `description`?

Reply:
no, conditional itself affects the task selection but the details here for import/include are different and are 'the override' we used to have in description.

Reply:
I really don't think that this is how attributes should be used. If two modules (in the same collection, or in core) use the same attribute name, they **really should have the same description**. The details can differ, **but not the description**.

Reply:
i was adding `details` had not yet refactored where to put the 'central' description, started to do that below